internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -874,6 +879,19 @@ func (c *ClusterManager) buildObservatoriumDexSecretResource() *k8sCoreV1.Secret
 	}
 }
 
+func (c *ClusterManager) buildDataplaneServiceaccountSecretResource(cluster *api.Cluster) *k8sCoreV1.Secret {
+	return &k8sCoreV1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      observabilityProxyServiceAccountSecretName,
+			Namespace: observabilityNamespace,
+		},
+		StringData: map[string]string{
+			"client_id":     cluster.ClientID,
+			"client_secret": cluster.ClientSecret,
+		},
```

Comment:
might be good to add the sso url where the account resides

Reply:
something akin to https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/25ee923c7994f5422d9652e98547f71ef4f5b586/internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go#L901 but the config would need to come from keycloak config.